app/about_us/page.js

```diff
@@ -22,7 +24,17 @@ const OurCrew = () => {
   // Use the pictures from /public/crew
   // Some inspiration ideas found in /data/inspiration_about_us
   return (
-    <p> ADD OUR CREW HERE </p>
+   <div>
+    <p>Meet the best space travelers and scientist in the world.</p>
```

Comment:
It is very important to use proper indentation (spaces/tabs) to show what is the parent and what are the children.
```
  <mainParent>
    <level2>
      <level3 />
    <level2>
    <level2Sibling>
    </level2Sibling>
  </mainParent>
```